src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -116,7 +116,7 @@ msrest==0.6.21
 msrestazure==0.6.4
 oauthlib==3.0.1
 packaging==21.3
-paramiko==2.6.0
+paramiko==2.10.1
```

Comment:
2.10.1 complies with https://github.com/Azure/azure-cli/blob/83c11a49f2e45fbee43a3aa57ec89a8e83169350/src/azure-cli-core/setup.py#L58

We should be good to go.